src/views/starter-projects/starter-projects.json

```diff
@@ -0,0 +1,206 @@
+{
+ "animation": [
+  {
+   "id": 10128067,
+   "type": "project",
+   "title": "Dance Party",
+   "thumbnailUrl": "https://cdn2.scratch.mit.edu/get_image/project/10128067_144x108.png",
+   "creator": "",
```

Comment:
The Carousel component requires a `username` for projects. It expects `item.author.username`. For now you can probably just use 'Scratchteam' for all of them:
```
"author": {"username": "Scratchteam"},
```